lmdeploy/serve/openai/api_server.py

```diff
@@ -111,11 +111,29 @@ def create_error_response(status: HTTPStatus, message: str):
 
 async def check_request(request) -> Optional[JSONResponse]:
     """Check if a request is valid."""
-    if request.model in get_model_list():
-        return
-    ret = create_error_response(
-        HTTPStatus.NOT_FOUND, f'The model `{request.model}` does not exist.')
-    return ret
+    if hasattr(request, 'model') and request.model not in get_model_list():
+        return create_error_response(
+            HTTPStatus.NOT_FOUND,
```

Comment:
The http status not found is designed to represent 404.
bad request should be ok here.